tbf/testcase_validation.py

```diff
@@ -419,7 +428,7 @@ def _get_compile_cmd(self,
         return cmd
 
     def compile(self, program_file, harness_file):
```

Comment:
How about adding a named argument `output_file_name='a.out'` to `compile`, and using that to provide a nice name for the final executable?

Reply:
I like it.  Will do.